packages/components/src/components/domainproperties/constants.ts

```diff
@@ -133,13 +133,8 @@ export const HIGHLIGHT_BLUE = '#2980B9';  // See $blue-border in variables.scss
 export const NOT_HIGHLIGHT_GRAY = '#999999';
 // export const HIGHLIGHT_BLUE = '#3495D2';
 
-export const LK_BASE_HELP_URL = "https://www.labkey.org/Documentation/wiki-page.view?name=";
-export const SM_BASE_HELP_URL = "https://www.labkey.org/SampleManagerDocs/wiki-page.view?name=";
-
-export const LK_URL_ENCODING_DOC = `${LK_BASE_HELP_URL}urlEncoding`;
-export const LK_DOMAIN_HELP_URL = `${LK_BASE_HELP_URL}fieldEditor`;
-export const LK_ASSAY_DESIGNER_HELP_URL = `${LK_BASE_HELP_URL}defineAssaySchema`;
-export const SAMPLE_TYPE_HELP_ANCHOR = "propertyFields#sampleType";  //Needs to be context/App aware
+// export const LK_BASE_HELP_URL = "https://www.labkey.org/Documentation/wiki-page.view?name=";
+// export const SM_BASE_HELP_URL = "https://www.labkey.org/SampleManagerDocs/wiki-page.view?name=";
```

Comment:
Remove these comment-out lines?

Reply:
I was leaving them in there in case they were useful for the dev when we do the SM vs LabKey doc link conditional logic.